torch/quantization/observer.py

```diff
@@ -985,6 +985,36 @@ def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
         super(HistogramObserver, self)._load_from_state_dict(state_dict, prefix, local_metadata, strict,
                                                              missing_keys, unexpected_keys, error_msgs)
 
+class Fp16Observer(ObserverBase):
```

Comment:
this seems to be the same as min max observer except for the warning, can we just use min max observer and put warning somewhere else?

Reply:
We don't have calculate_qparams and other methods defined for this observer. Also, feel that from user standpoint it might be better to separate this from MinMax observer since that actually observes the values to calculate the qparams. I can add a docblock to better explain the usage of this.

Reply:
I see, since we just use min_val/max_val for warning, can we remove it?

Reply:
If we can remove the warning then we can just use NoOp observer for this

Reply:
Having the warning is useful for users to know if their weight values might be getting saturated and it can tell them that they might need to update the model. So users can just insert the observers and run the model to check for potential overflow issues.

Reply:
can this be done during convert though? we only have one user facing API quantize_dynamic_jit